### PR TITLE
Update accessibility statement

### PR DESCRIPTION
**Pull request by gemmadallmandfe**
### Trello card
https://trello.com/c/9lR3X6AR

### Context
We need to update our accessibility statement following the DAC accessibility audit and associated changes.

### CODE REVIEW

File: app/views/pages/accessibility.html.erb [hunk 1]
```diff
@@ -4,99 +4,78 @@
...
     <p>
-      We want as many people as possible to be able to use this service. For
+      This website is run by the Department for Education. We want as many people as possible to be able to use this website. For
       example, that means you should be able to:
     </p>
     <ul class="govuk-list govuk-list--bullet">
       <li>change colours, contrast levels and fonts</li>
       <li>zoom in up to 300% without the text spilling off the screen</li>
-      <li>navigate most of the service using just a keyboard and speech recognition software</li>
+      <li>navigate most of the service using just a keyboard</li>
```
Comment (gemmadallmandfe) [inline app/views/pages/accessibility.html.erb:19]:
```suggestion
      <li>navigate most of the website using just a keyboard</li>
```

File: app/views/pages/accessibility.html.erb [hunk 2]
```diff
@@ -4,99 +4,78 @@
...
+      <li>navigate most of the service using speech recognition software</li>
```
Comment (gemmadallmandfe) [inline app/views/pages/accessibility.html.erb:20]:
```suggestion
      <li>navigate most of the website using speech recognition software</li>
```

File: app/views/pages/accessibility.html.erb [hunk 3]
```diff
@@ -4,99 +4,78 @@
...
+    <h2 class="govuk-heading-m">Reporting accessibility problems with this website</h2>
     <p>
-      We're always looking to improve the accessibility of this service.
+      We're always looking to improve the accessibility of this website.
     </p>
     <p>
       If you find any problems that are not listed on this page or think we're
-      not meeting accessibility requirements email
+      not meeting accessibility requirements, contact our helpdesk by emailing
       <a href="mailto:getintoteaching.helpdesk@education.gov.uk">getintoteaching.helpdesk@education.gov.uk</a>
```
Comment (gemmadallmandfe) [inline app/views/pages/accessibility.html.erb:49]:
```suggestion
      <a href="mailto:getintoteaching.helpdesk@education.gov.uk">getintoteaching.helpdesk@education.gov.uk</a>.
```

**Comment by gemmadallmandfe** [comment]
@ethax-ross Could you delete the existing DAC report as part of this PR please?

**Comment by gemmadallmandfe** [comment]
@GayeCord Let me know if you are happy with this

**Comment by GayeCord** [comment]
I'm happy! Fully compliant!

**Review by ethax-ross — DISMISSED** [review]

**Review by ethax-ross — APPROVED** [review]

**Pull request merged**